integration-tests/pom.xml

```diff
@@ -1,4 +1,4 @@
-<!-- Copyright (c) 2020, 2021, Oracle and/or its affiliates.
+<!-- Copyright (c) 2020, 2021, 2022, Oracle and/or its affiliates.
```

Comment:
It should be 2020, 2022,

Reply:
? why no 2021?

Reply:
Can you please follow the Corporate instruction  as posted by Ryan  in slack channel 
https://proddev-paas-fmw.slack.com/archives/C998UPSCT/p1641489147003000

Reply:
Changed